src/extension.ts

```diff
@@ -215,6 +224,15 @@ export function activate(context: vscode.ExtensionContext): void {
                     if (newApp.id === node.id) {
                         // if the node selected for deployment is the same newly created nodes, stifle the confirmDeployment dialog
                         confirmDeployment = false;
+                        newApp.treeItem.client.getSiteConfig().then(
+                            (createdAppConfig: SiteConfigResource) => {
+                                this.properties.os = createdAppConfig.linuxFxVersion ? 'Linux' : 'Windows';
+                                this.properties.runtime = createdAppConfig.linuxFxVersion ? createdAppConfig.linuxFxVersion : 'undefined';
```

Comment:
nit: I'd prefer properties.linuxFxVersion to be consistent with other telemetry: https://github.com/Microsoft/vscode-azuretools/blob/master/appservice/src/deploy/deploy.ts#L27

Reply:
Since this happens for every deploy action, having another property might make it easier to determine if it's the runtime for creation (so we can count created web apps) vs. just the runtime from a deployment-- not sure if that matters or not as we could filter on the createFromDeploy property?

Reply:
Yeah I was assuming we'd filter on your 'createdFromDeploy' property